blockchain/common_test.go

```diff
@@ -482,19 +515,94 @@ func newChaingenHarness(t *testing.T, params *chaincfg.Params, dbName string) (*
 		t.Fatalf("Failed to create generator: %v", err)
 	}
 
-	// Create a new database and chain instance to run tests against.
-	chain, teardownFunc, err := chainSetup(dbName, params)
+	return newChaingenHarnessWithGen(t, dbName, &g)
+}
+
+// AcceptHeader processes the block header associated with the given name in the
+// harness generator and expects it to be accepted, but not necessarily to the
+// main chain.  It also ensures the underlying block index is consistent with
+// the result.
+func (g *chaingenHarness) AcceptHeader(blockName string) {
+	g.t.Helper()
+
+	header := &g.BlockByName(blockName).Header
+	blockHash := header.BlockHash()
+	blockHeight := header.Height
+	g.t.Logf("Testing accept block header %q (hash %s, height %d)", blockName,
+		blockHash, blockHeight)
+
+	// Determine is the header is already known before attempting to process it.
```

Comment:
s / Determine is the header is... / Determine the header is...

Reply:
Changed to `Determine if the header ....` and also fixed another case of the same.